### DISCUSSION TITLE
프로젝트 flow 의견을 나누고 싶어요~

### DISCUSSION THREAD

**Discussion by jinux127** [heading1/WYLSBingsu#15] [Ideas]
1. login or register
2. 본인 빙수 url 생성
3. 친구들에게 공유
4. 익명으로 작성
5. 로그인 후 본인 빙수 확인

이라고 생각해봤는데 이게 맞나요??

**Comment by Circlewee** [comment]
4. 익명으로 작성(로그인 필수)라고 명세해주면 좋을것 같습니다

**Reply by Circlewee** [reply]
나머지는 맞아요!

**Reply by jinux127** [reply]
친구의 url에 들어가서 작성할때 로그인을 안하는게 사용자입장에서 덜 번거로울 것 같아서 로그인을 안하는 방향은 어떠세용?

**Reply by Circlewee** [reply]
세부 기획때 다시 얘기해보시죠!~

**Comment by sokojh** [comment]
네 동의합니다 ㅎㅎ
